p2p/p2p-core/src/lib.rs

```diff
@@ -89,13 +107,16 @@ pub trait NetZoneAddress:
     /// which for hidden services could just be the address it self but for clear net addresses will
     /// be the IP address.
     /// TODO: IP zone banning?
+    /// TODO: rename this to Host.
```

Comment:
Formatting for `cargo doc`.
```suggestion
    /// be the IP address.
    ///
    /// - TODO: IP zone banning?
    /// - TODO: rename this to Host.
```